src/civis/futures.py

```diff
@@ -39,14 +39,6 @@ class CivisFuture(PollableResult):
         first time. If ``False``, it will wait the number of seconds specified
         in `polling_interval` from object creation before polling.
 
-    Attributes
-    ----------
-    job_id : int
-        First element of the tuple given to `poller_args`
-    run_id : int or None
-        Second element of the tuple given to `poller_args`
-        (`None` if the poller function does not require a run ID)
-
```

Comment:
I added this at #290, but it looks like I should've added the docstrings to the respective `property` methods and let Sphinx handle the rest. After this PR, the `CivisFuture` doc will look correct:

<img width="638" alt="Screenshot 2024-05-16 at 11 07 05 AM" src="https://github.com/civisanalytics/civis-python/assets/86482098/b7659c82-0bd1-454d-8444-72759e00b7e0">